src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -106,6 +106,9 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
     hasEmailSettingsPanelExpanded: false,
   };
 
+  // to get the original session model on discard changes
+  feedbackSessionModels: Map<string, FeedbackSession> = new Map();
```

Comment:
Do you need this to be a `Map`? There should only be one feedback session involved in this component.

Reply:
On second look it doesn't need a `Map`, I was simply following the pattern for the `feedbackQuestionModels`.

So, I removed the `Map` and instead initialized `FeedbackSession` with empty strings and zeros, similar to `newQuestionEditFormModel`. Thanks for the feedback!